### PR TITLE
My Institute

### PR DESCRIPTION
**Pull request by mandanshubham**

### CODE REVIEW

File: GuruCool_Mobile/My Institute/Widgets/dialogBoxOne.dart [hunk 1]
```diff
@@ -0,0 +1,145 @@
+import 'package:flutter/material.dart';
+import 'package:my_institute/Utils/SizeConfig.dart';
+import '../Utils/constants.dart';
+
+void dialogBoxOne(BuildContext context) {
+  showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        SizeConfig().init(context);
```
Comment (rkishan516) [inline GuruCool_Mobile/My Institute/Widgets/dialogBoxOne.dart]: Make this dailog stateless widget

Comment (mandanshubham) [reply GuruCool_Mobile/My Institute/Widgets/dialogBoxOne.dart]: all the four of them?

**Review by rkishan516 — CHANGES_REQUESTED** [review]
Student and admin quiz has been already merged, remove them from here
Dialog should be widget, and rather than calling a function, directly call showDialog and then builder should return the Dialog widget

File: GuruCool_Mobile/My Institute/Widgets/dialogBoxOne.dart (hunk 1 above)
Comment (rkishan516) [reply GuruCool_Mobile/My Institute/Widgets/dialogBoxOne.dart]: Yes

**Review by rkishan516 — COMMENTED** [review]
@mandanshubham sync the repo or delete quiz section

**Pull request closed without merging**